helm-charts/opentelemetry-collector/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
```

Comment:
This condition should be combined with that of the standalone collector being true or we can wind up with a daemonset and an hpa.

Reply:
thanks. addressed